AWSClientRuntime/Tests/Http/AWSUserAgentMetadataTests.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import AWSClientRuntime
+
+class AWSUseragentMetadataTests: XCTestCase {
```

Comment:
were you not able to find the existing user agent tests?

Reply:
Ah, I didn't notice there were already UserAgentTests inside of a folder called "UserAgent" -- considering the source files were located in a folder called `Http` I was expecting the accompanying unit tests would be there as well -- i'll move the test over. to be consistent